rustls/src/client/builder.rs

```diff
@@ -68,40 +68,6 @@ impl ConfigBuilder<ClientConfig, WantsVerifier> {
     }
 }
 
-/// Container for unsafe APIs
```

Comment:
I prefer the old order... the module itself is `pub(super)` but its contents are `pub` (so I don't find the argument in the commit message convincing), and it fits in the flow of the module in the sense of the method just above it returning the `DangerousClientBuilder` and `DangerousClientBuilder::with_custom_certificate_verifier()` yielding `ConfigBuilder<ClientConfig, WantsClientCert>` where `WantsClientCert` is the item defined immediately below.